soccer/NewRefereeModule.cpp

```diff
@@ -106,11 +106,11 @@ static const int KickVerifyTime_ms = 250;
 // the ref halts/stops, make this false
 static const bool CancelBallPlaceOnHalt = true;
 
-NewRefereeModule::NewRefereeModule(SystemState& state)
+NewRefereeModule::NewRefereeModule(Context* context)
```

Comment:
Also maybe make this a const pointer, and then add a using declaration or something so instead of typing `Context* const context` all the time you can do something like `ContextPtr context` or something?

Reply:
There's really no point in making the parameter a const pointer - it doesn't really add any information to the interface IMO.

Reply:
Maybe not here but would it be worth at places where you're storing the pointer, ie. `Context* _context`?

Reply:
Hmm, that's true. I can go through and do that.